spec/javascript/components/mappers/domainToApi.test.js

```diff
@@ -0,0 +1,71 @@
+// TODO: Expand these tests to use fixtures, test all fields in application.
```

Comment:
I'd like to follow-up this PR with another one that overhauls how we do a lot of our testing